README.md

```diff
@@ -98,4 +100,40 @@ We can extend the concept of the `as` syntax by generalizing the right hand side
 import value from "module" as {key1: "value1", key2: [1, 2, 3]};
 ```
 
-This would allow module attributes to scale up to multiple pieces of data, not privileging a single attribute.
+This would allow module attributes to scale to support a variety of metadata. It is currently unclear if the generalized constant form will be pursued in this proposal or in a follow up. In either case the addition of the generalized constant form is not hard to design, specify and implement on top of the existing [spec text outline](https://tc39.github.io/proposal-module-attributes/). 
+
+## FAQ
+
+### Why not out of band?
+
+Why not both? The champions of this proposal think that exploring both an in- and out of band solutions to this problem are desirable. Depending on the environment in which JavaScript is being executed in either approach could be seen as desirable.
+
+While an in band solution is more verbose, it is also more straightforward for developers to adopt. For smaller projects developers do not need to create an extra file by hand. For large project with many dependencies developers will not have to worry about creating a large manifest by compiling the metadata of all of their dependencies. Module authors will also not have to worry about shipping a manifest in order for consumers to be able to run their modules.
+
+The [import-maps proposal](https://github.com/WICG/import-maps) is a great example of an out of band manifest that could be created at installation time, but its focus is on modules and at this time all necessary information to generate an application's import map could be found in the package.json of its dependencies.
```

Comment:
I'm confused by this paragraph. How does this relate to the rest?

Nit: s/its/it's/

Reply:
dropped paragraph